src/components/checkbox/checkbox.component.spec.ts

```diff
@@ -132,6 +132,18 @@ describe('Checkbox Component', () => {
         expect(touchedCallbackSpy).not.toHaveBeenCalled();
     });
 
+    it('should add a required attribute to the input when required is true', async () => {
+        component.required = true;
+
+        fixture.detectChanges();
+        await fixture.whenStable();
```

Comment:
I don't think that whenStable is needed here or for any of these tests, please confirm.